render/data/targets.go

```diff
@@ -4,12 +4,16 @@ import (
 	"fmt"
 	"time"
 
+	v3pb "github.com/go-graphite/protocol/carbonapi_v3_pb"
 	"github.com/lomik/graphite-clickhouse/config"
 	"github.com/lomik/graphite-clickhouse/helper/rollup"
 	"github.com/lomik/graphite-clickhouse/metrics"
 	"github.com/lomik/graphite-clickhouse/pkg/alias"
 )
 
+const GraphiteConsolidationFunction = "consolidateBy"
```

Comment:
we can make it private, taking into account that it is used within one package

Reply:
Done